.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,3 +10,6 @@ PR Checklist
  - [ ] Docs?
  - [ ] CHANGELOG updated?
  - [ ] Issue raised/referenced?
+
+## Developers certificate of origin
+- [ ] I certify that this contribution is covered by the LGPLv2.1+ license as defined [here](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).
```

Comment:
I am not a fan of "here" links
```suggestion
- [ ] I certify that this contribution is covered by the LGPLv2.1+ license as defined in our [LICENSE](https://github.com/MDAnalysis/mdanalysis/blob/develop/LICENSE) and adheres to the [**Developer Certificate of Origin**](https://developercertificate.org/).
```

Reply:
Done in [308e33e](https://github.com/MDAnalysis/mdanalysis/pull/4219/commits/308e33ef48c48e4bc06f51f9a3d7678ff771efa0)